collection/alexander.json

```diff
@@ -13,7 +13,8 @@
 		"jpn": "アレクサンダー",
 		"ita": "alessandro",
 		"tha": "อเล็กซานเดอร์",
-		"heb": "אלכסנדר"
+		"heb": "אלכסנדר",
+		"ro" : "Alexandru
```

Comment:
Hey @hackmajoris, thanks for you contribution.
Unfortunately this can't be merged as it's failing the tests. Could you please make sure you use the ISO-639-3 language codes for translations? Assuming this is a translation for Romanian, the 3 char code would be `ron`.
Also the supplied translations need to be all in lowercase characters.
In addition to that it seems like you are missing a closing `"` after the translation.

Just tag me here once you have fixed these issues and I'll happily merge this for you 😄